### PR TITLE
Improve docs for namespace scenarios

### PR DESCRIPTION
**Pull request by chaitanyaenr**

### CODE REVIEW

**Comment by paigerube14** [comment]
Need updates for this issue on kraken: https://github.com/cloud-bulldozer/kraken/issues/191

**Comment by paigerube14** [comment]
rerun

**Comment by paigerube14** [comment]
Probably need to fix this on the kraken side

```
Traceback (most recent call last):
  File "run_kraken.py", line 288, in <module>
    main(options.cfg)
  File "run_kraken.py", line 202, in main
    scenarios_list, config, wait_duration, failed_post_scenarios, kubeconfig_path
  File "/root/kraken/kraken/namespace_actions/common_namespace_functions.py", line 24, in run
    if scenario_label.strip() != "":
AttributeError: 'NoneType' object has no attribute 'strip'
```

File: namespace-scenarios/env.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Vars and respective defaults
 export ACTION=${ACTION:="delete"}
-export LABEL_SELECTOR=${LABEL_SELECTOR:="kubernetes.io/metadata.name=openshift-etcd"}
+export LABEL_SELECTOR=${LABEL_SELECTOR:=""}
```
Comment (paigerube14) [inline namespace-scenarios/env.sh:5]: what if we set label like the following? 

export LABEL_SELECTOR=${LABEL_SELECTOR:="''"}

Comment (chaitanyaenr) [reply namespace-scenarios/env.sh]: Fixed it, thanks.

File: docs/namespace-scenarios.md [hunk 2]
```diff
@@ -32,11 +32,10 @@ ex.)
 Parameter               | Description                                                           | Default
 ----------------------- | -----------------------------------------------------------------     | ------------------------------------ |
 ACTION                  | Action to take on the namespace                                       | delete                                  |
-LABEL_SELECTOR          | Label of the namspace to target                                       |                          |
-OBJECT_NAME             | List of the names of pods or nodes you want to skew ( optional parameter )                   | []                                   |
-NAMESPACE               | Namespace of the pods you want to skew, need to be set only if setting a specific pod name | ""                   |
+LABEL_SELECTOR          | Label of the namespace to target. Set this parameter only if NAMESPACE is not set                                       |     ""                     |
+NAMESPACE               | Name of the namespace you want to target. Set this parameter only if LABEL_SELECTOR is not set  | ""                   |
```
Comment (paigerube14) [inline docs/namespace-scenarios.md:36]: default here is actually "openshift-etcd", right?

File: docs/namespace-scenarios.md [hunk 3]
```diff
@@ -32,11 +32,10 @@ ex.)
...
 SLEEP                   | Number of seconds to wait before polling to see if namespace exists again         | 15                                    |
-RUNS                    | Number of namespaces to take the action on in each scenario           | 1                                    |
+RUNS                    | Number of runs to execute the action           | 1                                    |
```
Comment (paigerube14) [inline docs/namespace-scenarios.md:38]: can we also add in delete_count for this scenario

https://github.com/cloud-bulldozer/kraken/blob/master/docs/namespace_scenarios.md#delete-namespace-scenarios

**Comment by paigerube14** [comment]
Couple of doc nit picks but code runs well

File: docs/namespace-scenarios.md (hunk 2 above)
Comment (chaitanyaenr) [reply docs/namespace-scenarios.md]: Fixed it, thanks.

**Review by paigerube14 — APPROVED** [review]
LGTM

**Pull request merged**